test/response/api/v1/attendances/get.json

```diff
@@ -1,337 +1,504 @@
 {
   "items": [
     {
+      "id": 41761,
+      "team": {
+        "id": 3,
+        "href": "http://amam.sample.com/api/v1/teams/3"
+      },
+      "member": {
+        "id": "100606116",
+        "href": "http://amam.sample.com/api/v1/members/100606116"
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "http://amam.sample.com/api/v1/attendance_types/5"
+      },
+      "attendanceReason": {
+        "id": 6,
+        "href": "http://amam.sample.com/api/v1/attendance_reasons/6"
+      },
+      "remarks": "中国",
+      "createdAt": "2016-08-25T19:01:19.000+09:00",
+      "updatedAt": null,
+      "date": "2016-12-26",
```

Comment:
It depends on UI side but could we show start and end datetime considering time zone? It's hard for us to know accurate arrival time at the office for not Tokyo office members.